tests/test_matrix_props/test_is_permutation.py

```diff
@@ -0,0 +1,34 @@
+"""Test is_permutation."""
+import numpy as np
+
+from toqito.matrix_props import is_permutation

```

Comment:
Your import statement here is fine, but since you did not add 

`from toqito.matrix_props.is_permutation import is_permutation` 

in the `matrix_props/__init__.py`, this won't work.